fili-core/src/main/java/com/yahoo/bard/webservice/sql/helper/DatabaseHelper.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.sql.helper;
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.JDBCType;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+/**
+ * Various functions to help with interacting with a database.
+ */
+public class DatabaseHelper {
+    private static final int DATA_TYPE = 5;
+    private static final int COLUMN_NAME = 4;
+
+    /**
+     * Private constructor - all methods static.
+     */
+    private DatabaseHelper() {
+    }
+
+    /**
+     * Finds the name of the Timestamp column in a sql database.
+     *
+     * @param connection  The connection to the database.
+     * @param schema  The name of the schema (i.e. "PUBLIC").
+     * @param table  The name of the table (i.e. "table").
+     *
+     * @return the name of the timestamp column.
+     *
+     * @throws SQLException if failed while reading database.
+     * @throws IllegalStateException if no {@link JDBCType#TIMESTAMP} column could be found.
+     */
+    public static String getTimestampColumn(Connection connection, String schema, String table)
```

Comment:
While it would be nice to require that the Timestamp column is specified in the application config, it would get pretty difficult to set this up to work with multiple tables (having different names for their timestamp columns)

This might be better suited using the AbstractBinderFactory? @archolewa 
This could also take the role of specifying the schema for a table